### PR TITLE
Steam 862/planned course viz

### PR DESCRIPTION
**Pull request by Dtphelan1**
Creates a mapper and a visualizer for Planned Course data based on [this profile](https://build.fhir.org/ig/HL7/codex-radiation-therapy/StructureDefinition-codexrt-radiotherapy-planned-course.html). As for the data elements visualized, I mirror the Course Summary visualization. 

A few other small changes included in this PR: 
- Added a link to the FHIRpath docs;
- Added new patients to the default list of queries; these will have more complete records;
- `src/components/DataView/DataView.js`: Made the comments more accurate
- Increases the max-height of the patient query list;
- `src/lib/fetchingUtils.js`: Turned some lets into consts

### CODE REVIEW

File: src/lib/mappingUtils.js [hunk 1]
```diff
@@ -22,28 +22,28 @@ function mapPatient(patient) {
  * @returns {Object} Returns an object with key/value pairs of data to display in the table
  */
 function mapCourseSummary(procedure) {
-  let summary = fhirpath.evaluate(
+  const summary = fhirpath.evaluate(
     procedure,
-    "Bundle.entry.where(resource.meta.profile.first() = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-course-summary').resource"
```
Comment (julianxcarter) [inline src/lib/mappingUtils.js:27]: Neat!

File: src/lib/mappingUtils.js [hunk 2]
```diff
@@ -107,16 +107,80 @@ function mapPhase(procedure) {
...
+        plannedCourse,
+        "ServiceRequest.extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality-and-technique').extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality').valueCodeableConcept.coding"
+      )
+      .map((modality) => modality.display)[0];
+    output["Techniques"] = fhirpath
+      .evaluate(
+        plannedCourse,
+        "ServiceRequest.extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality-and-technique').extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-technique').valueCodeableConcept.coding"
+      )
+      .map((technique) => technique.display)
```
Comment (julianxcarter) [inline src/lib/mappingUtils.js]: This feels somewhat nitpicky but do we want some kind of null guard in the case that `technique.display` isn't present on this coding? (This is a demo app after all so feel free to say no, this hasn't been a problem thus far.)

File: src/lib/mappingUtils.js [hunk 3]
```diff
@@ -107,16 +107,80 @@ function mapPhase(procedure) {
...
+        "ServiceRequest.extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-procedure-intent').valueCodeableConcept.single().coding"
+      )
+      .map((intent) => intent.display)[0];
+
+    output["Modalities"] = fhirpath
+      .evaluate(
+        plannedCourse,
+        "ServiceRequest.extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality-and-technique').extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality').valueCodeableConcept.coding"
+      )
+      .map((modality) => modality.display)[0];
```
Comment (julianxcarter) [inline src/lib/mappingUtils.js]: Similar comment here in regards to guarding against an undefined `display` value.

**Review by julianxcarter — COMMENTED** [review]
This looks good to me! Just two questions.

File: src/lib/mappingUtils.js (hunk 3 above)
Comment (Dtphelan1) [reply src/lib/mappingUtils.js]: lol why don't I just get the display off the coding in the fhir path....

File: src/lib/mappingUtils.js (hunk 2 above)
Comment (Dtphelan1) [reply src/lib/mappingUtils.js]: lol I should just get the display off the object in the fhir path

File: src/lib/mappingUtils.js (hunk 1 above)
Comment (Dtphelan1) [reply src/lib/mappingUtils.js:27]: Thanks! 😄

**Comment by Dtphelan1** [comment]
Also, from Michelle C: 
> Yes, basing it off the existing course summary and treated phase is the right approach.

So the selected fields should be fine for the initial pass.

**Review by julianxcarter — APPROVED** [review]
Looks good to me! Merging since Michelle has given her blessing.

**Pull request merged**